docs/source/javascripts/documentation.js

```diff
@@ -51,3 +52,23 @@ function sidebarFollowScroll() {
     }
   });
 }
+
+// The Following code is used to set active items
+// On the documentation sidebar depending on the
+// clicked item
+
+  let linksContainer = document.querySelector('#sidebar ul');
```

Comment:
Sure, by the way, kudos to @proudlygeek for helping me A LOT with this function. :sparkles: